pkg/volume/flexvolume/plugin.go

```diff
@@ -64,44 +65,25 @@ func NewFlexVolumePlugin(pluginDir, name string) (volume.VolumePlugin, error) {
 		unsupportedCommands: []string{},
 	}
 
-	// Check whether the plugin is attachable.
-	ok, err := isAttachable(flexPlugin)
+	call := flexPlugin.NewDriverCall(initCmd)
```

Comment:
We should make it very explicit that driver initialization is done when the plugin is instantiated, instead of during the VolumePlugin.Init() call like one might expect.